examples/tensorrt_llm/common/protocol.py

```diff
@@ -102,3 +129,4 @@ class TRTLLMWorkerRequest(BaseModel):
     annotations: List[str] = Field(default_factory=list)
     estimated_prefix_hit_num_blocks: Optional[int] = None
     disaggregated_params: Optional[DisaggregatedParams] = Field(default=None)
+    kv_cache_retention_config: Optional[KvCacheRetentionConfig] = Field(default=None)
```

Comment:
Is the `TRTLLMWorkerRequest` here identical to `PreprocessedRequest` in `preprocessor.rs`?

Reply:
It is mostly the same in addition to the optional disaggregated_params field.